cl is needed at runtime for function remove-if-not (and maybe others).

--- rudel-0.2-4-orig/rudel-backend.el
+++ rudel-0.2-4/rudel-backend.el
@@ -46,9 +46,7 @@
 ;;; Code:
 ;;
 
-(eval-when-compile
-  (require 'cl))
-
+(require 'cl)
 (require 'eieio)
 
 
--- rudel-0.2-4-orig/rudel-overlay.el
+++ rudel-0.2-4/rudel-overlay.el
@@ -35,9 +35,7 @@
 ;;
 
 (require 'custom)
-
-(eval-when-compile
-  (require 'cl))
+(require 'cl)
 
 
 ;;; Rudel overlay faces
